packages/react/src/experimental/RichText/RichTextEditor/Enhance/index.tsx

```diff
@@ -41,7 +41,8 @@ const EnhanceActivator = ({
   const enhanceButtonRef = useRef<HTMLButtonElement>(null)
   const [open, setOpen] = useState(false)
 
-  const handleEnhanceClick = () => {
+  const handleEnhanceClick = (e?: React.MouseEvent) => {
```

Comment:
I think there is always an event passed to the handler, no need to make it optional